src/Windows/Core/Impl/Shell/ICoreShell.cs

```diff
@@ -12,7 +11,12 @@ namespace Microsoft.Common.Core.Shell {
     /// composition container, export provider, global VS IDE
     /// services and so on.
     /// </summary>
-    public interface ICoreShell : ICompositionCatalog {
+    public interface ICoreShell : IIdleTimeService, ICompositionCatalog {
+        /// <summary>
+        /// Application-global services access
+        /// </summary>
+        IServiceContainer GlobalServices { get; }
```

Comment:
Yes, current `Services` is temporary. I just want bite-size PRs. Next time there will be only one.